typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -296,14 +296,14 @@ export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
 
         const coarseData = coarsenWells(data);
 
-        const doRefine =
-            typeof refine === "number" ? refine > 1 : (refine as boolean);
+        const isDataTransformSet = this.props.dataTransform !== null;
```

Comment:
Would it work to set the default value of `dataTransform` to coarsenWells instead?

Reply:
Not really as dataTransform may do anything but here we need a coarse version for the simplified rendering,

Reply:
Fixed in https://github.com/equinor/webviz-subsurface-components/pull/2463/commits/5f8947552a7bfdf82a1534d0a8f0fbe49168ddf4